docs/knowledge_base/tips/3rd_party_components/index.md

```diff
@@ -71,28 +72,20 @@ def expose_folium(map):
 In this code, the function *expose_folium()* converts a Folium object (*map*) to HTML, a 
 mandatory step in integrating any third-party component into your application.
 
-The *map* object is transformed into HTML, and then we proceed to define two callback 
-functions, namely, *on_user_content()* and *on_init()*. The *on_init()* callback serves 
-as a fundamental Taipy callback that is triggered when a new user connects to the 
-application. On the other hand, the *on_user_content()* function is designed to return 
-the HTML content to be rendered, while the *get_user_content_url()* function is invoked 
-to obtain the URL for rendering the HTML content.
+We now proceed to register a function for the object type we want to display. The code below 
+means that when Taipy encounters a *Map* type in the *content* property, it will use the 
+*expose_folium* function to convert it to HTML and display it on the page.
 
 ```python
-uc_url = None
+from folium.folium import Map
```

Comment:
Is this exactly the content of example.py?
If so, you can use the following syntax to include the code directly:

```python linenums="1"
{%
include-markdown "./example.py"
comments=true
%}
```